src/components/createOnyxContext.tsx

```diff
@@ -3,42 +3,40 @@ import type {ComponentType, ForwardedRef, ForwardRefExoticComponent, PropsWithou
 import React, {createContext, forwardRef, useContext} from 'react';
 import {withOnyx} from 'react-native-onyx';
 import getComponentDisplayName from '@libs/getComponentDisplayName';
-import type {OnyxCollectionKey, OnyxKey, OnyxKeyValue, OnyxValues} from '@src/ONYXKEYS';
+import type {OnyxKey, OnyxValue, OnyxValues} from '@src/ONYXKEYS';
 import type ChildrenProps from '@src/types/utils/ChildrenProps';
 
-type OnyxKeys = (OnyxKey | OnyxCollectionKey) & keyof OnyxValues;
```

Comment:
Previously we needed to have `keyof OnyxValues` here so that typescript doesn't yell. Now `OnyxKey` type is derived from `OnyxValues` so it is cleaner now.